### ISSUE TITLE
"Modify Treatment plan" functionality

### ISSUE THREAD

**Issue by OlenkaP** [RiLights/aligntime#41]
The idea is that users should be able to modify answers to some of the welcome screen questings in case their treatment plan is gonna change along the way.

![image](https://user-images.githubusercontent.com/52235757/72689852-d2636680-3b7a-11ea-988d-f7878cc5f956.png)

Total number of aligners can bee modified down the track if let's say user's treatment plan has changed and not they require 74 aligners instead of 77 (range for these values is still from 1 to 200)

Treatment start date can be modified as well, using out standard date picker functionality ( calendar can go back to 3 years from the current system date)

There there will be number of days for each aligner with the ability to modify them:
- the default value will be populated from the question user answered at the welcome screen _Number of days for each aligner?_ (i.e. 7 days), but then users should be able to modify those values for each aligner right up to total number of selected aligners (i.e. 74) 
- there should be the ability to scroll till the last aligner (ie #74) at the bottom of the page
- ranges are from 1 day to 31 days

**Comment by RiLights** [comment]
Hey @OlenkaP,

Why "Modify Treatment plan" has just a few options from the original setup (e.g Aligner number you are wearing now)?

**Comment by OlenkaP** [comment]
@RiLights this is very good question! Im not actually sure, why I've done it this way ....
it would be reasonable to 'Aligner you're wearing now field there as well'

However, we won't need ' Number of days for each aligner' as a separate field, as those values would be populated as a default value into each of the 'Aligner # ...' lines

**Comment by RiLights** [comment]
But maybe it make sense to have default  'Number of days for aligner'?  To not reasigne days for the each aligner.

**Comment by OlenkaP** [comment]
@RiLights yes, so definitely default 'Number of days for aligner' based on initial users answer on the welcome screen, cause this is unlikely to change in the future.
If we decide to show it as a separate line in the 'Modify Treatment plan' section, we need to make sure once default value is changed, it gets displayed into every Aligner #  line

For example:
- we add line that says 'Number of days for each aligner' and during welcome screen set up user selects '7';
- when they navigate to 'Modify Treatment plan' section each of this lines would be populated with 7 by default
![image](https://user-images.githubusercontent.com/52235757/74889483-1057e280-53e6-11ea-8b0a-9fb7841214ac.png)


- then if user decides to change it from 7 to 10 for all of them, they just modify line 'Number of days  fro each aligner' and that would auto populate the other fields with the current value, however we need to be cautious and make sure that  fields that get auto-populated with new default value are only the fields from Aligner # = current aligner (i.e. 29) and onward, rather than every single aligner.

![image](https://user-images.githubusercontent.com/52235757/74889756-dc30f180-53e6-11ea-80e0-fcef1149e3c4.png)

**Comment by RiLights** [comment]
As an experiment, I tried dynamic adjustment. Would be nice if you can take a look.
Build 6 should will be available for you soon.
